### ISSUE TITLE
Documentation: Unclear which engine(s) implement which nodes

### ISSUE THREAD

**Issue by hybridherbst** [KhronosGroup/glTF-InteractivityGraph-AuthoringTool#6]
My understanding was that there would be a set of "core" KHR_interactivity nodes that would have to be implemented by each engine claiming KHR_interactivity support. 

However, in the current implementation, e.g. the logging engine does not support animation nodes:
<img width="945" alt="image" src="https://github.com/KhronosGroup/glTF-InteractivityGraph-AuthoringTool/assets/2693840/1870ac39-33c7-47c0-acf3-9f94310d7b79">

I saw there have been recent changes from `lifecycle/*` to `event/*`, so maybe the Logging engine implementation just hasn't been updated yet.

**Comment by mattmacf98** [comment]
yes the logging engine hasn't been updated to support some 3D specific actions, I will probably implement them as a no-op log asp. For now to see the supported log and babylon engine nodes you can look at this function https://github.com/KhronosGroup/glTF-InteractivityGraph-AuthoringTool/blob/initial-work-merge/src/BasicBehaveEngine/BasicBehaveEngine.ts#L293-L374

ones that are just supported in babylon are found here https://github.com/KhronosGroup/glTF-InteractivityGraph-AuthoringTool/blob/initial-work-merge/src/BasicBehaveEngine/decorators/BabylonDecorator.ts#L45-L48
